package.json

```diff
@@ -62,6 +62,7 @@
     "babel-core": "^6.2.1",
     "babel-preset-es2015": "^6.1.18",
     "batch": "^0.5.1",
+    "fastly": "Financial-Times/fastly#v2.5.0",
```

Comment:
Ok, thanks.  I'm linking to a fixed tag so we should be OK with this one.  I'm not sure whether we are moving towards or away from CLI tools at the moment, with haikro on it's way out once we get a docker architecture we like, so for the moment i'd be pretty happy using a grunt task that fits in with our existing task architecture.

Also the fastly-tools thing appears not to support dry running a VCL without activating it.